ipsec-vpn/hier/usr/lib/python3/dist-packages/tests/test_ipsec_vpn.py

```diff
@@ -842,10 +845,14 @@ def test_082_any_remote_tunnel_ping(self):
                                               "event_type", "UNREACHABLE" )
         assert(found == False)
 
+        # removed added tunnel lists
+        ipsec_settings["tunnels"]["list"] = []
+        ipsec_settings = self._app.getSettings()
```

Comment:
To remove it we need to set the settings in app `self._app.setSettings(ipsec_settings)`